css/ToDo.css

```
@@ -0,0 +1,179 @@
+@charset "utf-8";
+@import url('https://fonts.googleapis.com/css2?family=Gugi&family=Noto+Sans+KR:wght@400;500&display=swap');
+
+
+/* reset */
+body,div,h1,label,input{ /* 텍스트 */
+    display: block;
+    font: normal 14px/1.5em 'Noto Sans KR',sans-serif;
+    text-align: center;
+}
```

전역 css 설정을 할때는 기본값으로 설정할때만 사용하며, 최소한으로 쓰는것을 추천합니다.
또한, body속성 또는 html 속성에 기본 폰트를 설정하면 나머지 요소들도 기본적으로 body의 속성을 상속받아오므로
아래와 같이 변경하는 것을 추천합니다.
```suggestion
body{ /* 텍스트 */
    font: normal 14px/1.5em 'Noto Sans KR',sans-serif;
}
```